sci-misc/boinc/boinc-7.2.44-r2.ebuild

```diff
@@ -118,6 +118,11 @@ src_install() {
 			newicon -s $s "${WORKDIR}"/boinc_${s}.png boinc.png
 		done
 		make_desktop_entry boincmgr "${PN}" "${PN}" "Math;Science" "Path=/var/lib/${PN}"
+
+		# Rename the desktop file to boincmgr.desktop to (hot)fix bug 599910
+		mv "${ED}"/usr/share/applications/boincmgr-${PN}.desktop \
+		   "${ED}"/usr/share/applications/boincmgr.desktop || \
+		   die "Failed to rename desktop file"
```

Comment:
shorten this to
`mv "${ED%/}"/usr/share/applications/boincmgr{-${PN},}.desktop || die "Failed to rename desktop file"`
always use ${ED%/} due to double-slashing (last section of https://devmanual.gentoo.org/ebuild-writing/variables/)